pkg/codegen/internal/test/testdata/external-resource-schema/schema.json

```diff
@@ -148,18 +182,23 @@
       "packageReferences": {
         "Pulumi": "3.13",
         "Pulumi.Aws": "4.20",
+        "Pulumi.GoogleNative": "0.11.0",
         "Pulumi.Kubernetes": "3.7.0",
         "Pulumi.Random": "4.2.0"
       }
     },
     "go": {
       "generateResourceContainerTypes": true,
-      "generateExtraInputTypes": true
+      "generateExtraInputTypes": true,
+      "packageImportAliases": {
+        "github.com/pulumi/pulumi-google-native/sdk/go/google/iam/v1": "gcpiamv1"
```

Comment:
I'm still learning our approach to codegen tests... Would it make sense to adjust one of the existing resources to test this alias? Or do intend to have a separate test case for any feature?